src/main/java/net/masterthought/cucumber/Trends.java

```diff
@@ -233,4 +312,13 @@ private void fillMissingDurations() {
         System.arraycopy(durations, 0, extendedArray, buildNumbers.length - durations.length, durations.length);
         durations = extendedArray;
     }
+
+    private void fillMissingFeaturesDetail(){
```

Comment:
another method - looks like filling some gaps :(

Reply:
this gaps especially is from user that update to this version. They don't need to remove current trend json file. Just run this version and ```fillMissingFeaturesDetail``` will handle for build that doesn't have featureDetails